### DISCUSSION TITLE
Gitleaks-Action V2 Announcement

### DISCUSSION THREAD

**Discussion by zricethezav** [Security-Onion-Solutions/securityonion#8138] [Ideas] [answered]
Hi Security-Onion-Solutions!

I maintain gitleaks and we noticed that you are not pinning your gitleaks-action version, but rather [using gitleaks-action@master](https://github.com/Security-Onion-Solutions/securityonion/blob/master/.github/workflows/leaktest.yml#L15) and wanted to give you a heads up that we are planning to merge v2 to master soon, which is a breaking update. See more details [here](https://github.com/gitleaks/gitleaks-action#-announcement). By the way, we think you'll love [v2's features](https://github.com/gitleaks/gitleaks-action#v2-benefits). Let us know if you have any questions!

cc @jertel (I see you have contributed to this workflow)

**Comment by defensivedepth** [comment, marked as answer]
Thanks for the heads up @zricethezav! We will pin 1.6.0 for now.

Congrats on shipping v2!
